packages/transformers/js/src/JSTransformer.js

```diff
@@ -74,9 +76,16 @@ export default new Transformer({
     let ast = asset.ast;
     let code = await asset.getCode();
 
-    // Inline environment variables
-    if (!asset.env.isNode() && (ast.isDirty || ENV_RE.test(code))) {
-      walk.simple(ast.program, envVisitor, {asset, env: options.env});
+    // Inline process/ environment variables
+    if (
+      !asset.env.isNode() &&
```

Comment:
It still wouldn't run on `electron-renderer` because of this `isNode`.

Reply:
Good catch! The tests also complained after the splitting 👍